webservices/rest.py

```diff
@@ -217,6 +217,7 @@ def add_aggregate_resource(api, view, schedule, label):
 api.add_resource(download.DownloadView, '/download/<path:path>/')
 
 api.add_resource(legal.Search, '/legal/search/')
+api.add_resource(legal.Load, '/legal/load/')
```

Comment:
Per our code review session:  Let's change this URL to be something different since this is a write.